roles/glance/tasks/ceph_integration.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- name: fetch rados.py
+  get_url: url=https://raw.githubusercontent.com/ceph/ceph/v0.94.3/src/pybind/rados.py
+           dest=/opt/bbc/openstack-11.0-master/glance/lib/python2.7/site-packages/rados.py
+           owner=root
+           group=root
+           mode=0644
+
+- name: fetch rbd.py
+  get_url: url=https://raw.githubusercontent.com/ceph/ceph/v0.94.3/src/pybind/rbd.py
+           dest=/opt/bbc/openstack-11.0-master/glance/lib/python2.7/site-packages/rbd.py
```

Comment:
Same comments about using variables for venv path and for version of pybind to grab from.